src/renderer/widgets/CreateWallet/lib/create-multisig-utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Step } from './types';
+
+export const createMultisigUtils = {
+  isSignStep,
+  isSubmitStep,
+  isConfirmStep,
+  isNameNetworkStep,
+  isSignatoriesThresholdStep,
+  isNotFirstStep,
+};
+
+function isConfirmStep(step: Step) {
```

Comment:
@Asmadek I remember we had a talk between me, you and @sokolova-an about some utility function instead of step checker functions. Have anything been done regarding it and @Tbaut should use that function/approach?

Reply:
[isStep](https://github.com/novasamatech/nova-spektr/blob/a59f223e14813dab5ce2849650b0478c7cb211c0/src/renderer/shared/lib/utils/step.ts#L16) function can be used

Reply:
Thanks, I re-declared one since my steps are different, but used this approach none the less.